js/bittrex.js

```diff
@@ -1051,176 +1054,156 @@ module.exports = class bittrex extends Exchange {
-            // 'clineConditionalOrderId': 'f03d5e98-b5ac-48fb-8647-dd4db828a297', // STOP
+        const newOrder = {
+            'marketSymbol': market['id'],
+            'direction': side.toUpperCase (),
         };
-        let method = 'privatePostOrders';
-        if (stop || stopPrice) {
-            method = 'privatePostConditionalOrders';
-            const operand = this.safeString (params, 'operand');
-            if (operand === undefined) {
-                throw new ArgumentsRequired (this.id + ' createOrder() requires an operand parameter');
-            }
-            const trailingStopPercent = this.safeNumber (params, 'trailingStopPercent');
-            const orderToCreate = this.safeValue (params, 'orderToCreate');
-            const orderToCancel = this.safeValue (params, 'orderToCancel');
-            if (stopPrice === undefined) {
-                request['trailingStopPercent'] = this.priceToPrecision (symbol, trailingStopPercent);
-            }
-            if (orderToCreate) {
-                const isCeilingLimit = (uppercaseType === 'CEILING_LIMIT');
-                const isCeilingMarket = (uppercaseType === 'CEILING_MARKET');
-                const isCeilingOrder = isCeilingLimit || isCeilingMarket;
-                let ceiling = undefined;
-                let limit = undefined;
-                let timeInForce = undefined;
-                if (isCeilingOrder) {
-                    let cost = undefined;
-                    if (isCeilingLimit) {
-                        limit = this.priceToPrecision (symbol, price);
-                        cost = this.safeNumber2 (params, 'ceiling', 'cost', amount);
-                    } else if (isCeilingMarket) {
-                        cost = this.safeNumber2 (params, 'ceiling', 'cost');
-                        if (cost === undefined) {
-                            if (price === undefined) {
-                                cost = amount;
-                            } else {
-                                cost = amount * price;
-                            }
-                        }
-                    }
-                    ceiling = this.costToPrecision (symbol, cost);
-                    timeInForce = 'IMMEDIATE_OR_CANCEL';
+        const upperCaseType = type.toUpperCase ();
+        const isMarketOrder = upperCaseType === 'MARKET' || upperCaseType === 'CEILING_MARKET';
+        const isLimitOrder = upperCaseType === 'LIMIT' || upperCaseType === 'CEILING_LIMIT';
+        const stopLossPrice = this.safeNumber (params, 'stopLossPrice');
+        const takeProfitPrice = this.safeNumber (params, 'takeProfitPrice');
+        if (stopLossPrice !== undefined && takeProfitPrice !== undefined) {
+            throw new ExchangeError (this.id + ' createOrder() does not support both stopLossPrice and takeProfitPrice params, please only supply one');
+        }
+        const isStopLossOrder = (stopLossPrice !== undefined);
```

Comment:
@DoctorSlimm, I'm not sure if we should be calling them `stopOrder/takeProfit` they can be used for that, but we can be placing a regular trigger order without previous orders opened. @kroitor, what do you think?

https://bittrex.github.io/api/v3#/definitions/NewConditionalOrder

Reply:
I named them this way because they are trigger orders + directional to make it clear what they are. Otherwise we would need to decide on some way to unify a `direction` parameter for the trigger, for how it is crossed I think.

When the trigger order available to be made by the API is such that it executes when the prices crosses from above (for sell orders), i have been calling those stopLossPrice orders.

And if the trigger order available to be made by the API is such that it executes if the price rises above the trigger, I have been calling those takeProfit orders to make it clear.

I thought this was the decided convention for `trigger orders with direction` (type 2 I believe)

https://github.com/ccxt/ccxt/issues/12044#issuecomment-1139118302

Reply:
@DoctorSlimm I perfectly understand the reasoning and I'm not saying it's wrong, but idk, in my opinion, might be a bit misleading. 

Let's say you want to place a standalone trigger order, "stop loss" and "take profit" are not quite valid concepts, in my opinion, because you're not setting any real sl/tk cap, only deciding when the order should be "activated" if that makes sense

Reply:
@carlosmiei  you are right, let me know how you think we could otherwise let the users distinguish between direction, maybe some alternative parameters perhaps

Reply:
@DoctorSlimm, I don't have access, can you check how the Web UI handles it?